dashboard/app/controllers/pd/csf_certificate_controller.rb

```diff
@@ -0,0 +1,15 @@
+class Pd::CsfCertificateController < ApplicationController
+  before_action :authenticate_user!
+  load_resource :enrollment, class: 'Pd::Enrollment', find_by: :code, id_param: :enrollment_code
+
```

Comment:
please add some basic tests for this controller:
 - authenticated users can access
 - non-users cannot
 - valid enrollment code works
 - invalid enrollment code gets not found